src/cljs/org/broadinstitute/firecloud_ui/common/components.cljs

```diff
@@ -49,6 +49,18 @@
             "+"]])]))})
 
 
+(react/defc Checkbox
+  {:render
+   (fn [{:keys [props]}]
+     [:label {:style {:cursor "pointer"}}
+      [:input {:type "checkbox" :ref "check"
+               :style {:cursor "pointer"}}]
+      [:span {:style {:marginLeft "0.5ex"}} (:label props)]])
+   :checked?
```

Comment:
By convention, I usually put "public" methods before "override" methods and then "private" methods last. Not a big deal.